langlib/lang.decimal/src/main/ballerina/decimal.bal

```diff
@@ -110,6 +170,14 @@ public isolated function ceiling(decimal x) returns decimal = @java:Method {
 # - a FloatingPointTypeSuffix is not allowed
 # This is the inverse of function ``value:toString`` applied to an `decimal`.
 #
+# ```
+# decimal:fromString("0.2453") ⇒ 0.2453
+# 
+# decimal:fromString("-10") ⇒ -10
+# 
+# decimal:fromString("Hello") ⇒ error
```

Comment:
```suggestion
# decimal:fromString("") ⇒ error
```

Highly unlikely to use something like Hello?

Reply:
But this will give the behaviour directly rather than using `""`. wdyt?

Reply:
Similar reason as https://github.com/ballerina-platform/ballerina-lang/pull/38905#discussion_r1050351552.